packages/cli/src/commands/deploy.ts

```diff
@@ -127,20 +139,15 @@ export default class Deploy extends Command {
         new ServiceUploader(minioOptions),
       ),
     )
-    const processFiles = readdirSync(path, {
-      withFileTypes: true,
-    }).filter((x) => x.isFile() && x.name.match(PROCESS_REGEX))
-    const compilation = new Listr(
-      processFiles.map((file) => ({
-        title: `Compiling ${file.name}`,
-        task: async () => {
-          this.processesToDeploy.push(
-            await compiler.compile(join(path, file.name), envs),
-          )
-        },
-      })),
+
+    const liteflowFile = files.find((x) => existsSync(join(path, x)))
+    if (!liteflowFile) throw new Error('liteflow.yml file not found')
+
+    this.processesToDeploy = await compiler.compile(
+      join(path, liteflowFile),
```

Comment:
We should keep only system to reference the file:
  - either the first ARG of the command with fallback to a default value
  - either an optional flag with a default value (like serverless: https://www.serverless.com/framework/docs/providers/aws/cli-reference/deploy/)

```suggestion
      liteflowFile,
```

Reply:
I really don't like having the arg to the file. If I want to deploy a specific directory I don't want to have to put the file that is default. System like docker, I can build a directory but if I have something not "standard" then I have to add an extra flag. This is 2 different stuffs here, the path of the project you want to deploy and the file use to configure this deployment (that should be almost all the time the default)

Reply:
we should compare it to docker-compose witch only accept the file:
```
  docker-compose [-f <arg>...] [options] [--] [COMMAND] [ARGS...]
  docker-compose -h|--help

Options:
  -f, --file FILE             Specify an alternate compose file
                              (default: docker-compose.yml)
```

It cannot be compared to docker build because both the directory AND the dockerfile are important. The directory is the one that the stuff of the host side in the dockerfile will be referenced from.
In our case, we absolutely don't care about the path, only the app file (liteflow.yml) is important and all reference in this file should be relative to the file itself (what we implemented the other day)

Reply:
As discussed, we removed the flag and dir to simplify the cli for now, we can think about this later